main.js

```diff
@@ -0,0 +1,97 @@
+let startButton = document.getElementById('start');
+startButton.addEventListener('click', () => {
+  startButton.textContent = "Player X's turn"
+  startButton.disabled = true;
+});
+let gameBoard = document.getElementById('board');
+let playerTurn = 'X';
+let column1 = document.getElementsByClassName('col-1');
+let column2 = document.getElementsByClassName('col-2');
+let column3 = document.getElementsByClassName('col-3');
+let diag1 = document.getElementsByClassName('diag-1');
+let diag2 = document.getElementsByClassName('diag-2');
+let row1 = document.getElementById('row-1');
+let row2 = document.getElementById('row-2');
+let row3 = document.getElementById('row-3');
+
+function loopaX(x) {
+  for (let content of x) {
+    if (content.innerText === 'X') {
```

Comment:
It is not a good coding habit t create if statements with empty bodies with a full else statement. In addition, you should be checking innerHTML, not innerText. Lastly, there should be a return statement outside of the for loop that returns true if the loop runs without intteruptions. Here is a more concise structure of this loop:
```js
for(let content of x) {
   if(content.innerHTML !== 'X') {
     return false;
   }
}
return true
```